internal/pkg/ecslogging/service.go

```diff
@@ -44,20 +47,21 @@ type WriteLogEventsOpts struct {
 // The logging client is initialized from the given sess session.
 func NewServiceClient(sess *session.Session, app, env, svc string) *ServiceClient {
 	return &ServiceClient{
-		logGroupName: fmt.Sprintf(logGroupNamePattern, app, env, svc),
-		eventsGetter: cloudwatchlogs.New(sess),
-		w:            log.OutputWriter,
+		logGroupName:        fmt.Sprintf(fmtSvclogGroupName, app, env, svc),
+		logStreamNamePrefix: fmt.Sprintf(fmtSvcLogStreamPrefix, svc),
+		eventsGetter:        cloudwatchlogs.New(sess),
+		w:                   log.OutputWriter,
 	}
 }
 
 // WriteLogEvents writes service logs.
 func (s *ServiceClient) WriteLogEvents(opts WriteLogEventsOpts) error {
 	logEventsOpts := cloudwatchlogs.LogEventsOpts{
-		LogGroup:  s.logGroupName,
-		Limit:     aws.Int64(int64(opts.Limit)),
-		EndTime:   opts.EndTime,
-		StartTime: opts.StartTime,
-		// TODO: Add log filtering by task ID.
+		LogGroup:   s.logGroupName,
+		Limit:      aws.Int64(int64(opts.Limit)),
+		EndTime:    opts.EndTime,
+		StartTime:  opts.StartTime,
+		LogStreams: s.logStreams(opts.TaskIDs),
```

Comment:
nit: can you add a comment here saying that if `LogStreams` is `nil` then it retrieves logs from all streams